targets/xiwi

```diff
@@ -28,8 +28,27 @@ if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
     apt-mark unhold xserver-xorg-video-dummy$ltspackages || true 2>/dev/null
```

Comment:
Okay, two dumb questions on this:
1. Is there a reason why the driver has to overwrite the dummy driver? Why not install it as a different driver (crouton_dummy_drv.so perhaps?) and then change the X configuration to load that driver instead?
2.  Is there a reason why the patch is inlined in the shell script rather than sitting in a separate file? I was thinking of applying it as an overlay patch to the Gentoo package, which would also resolve the issue of the driver being overwritten.

Thanks!